app/src/main/java/com/zulip/android/util/Constants.java

```diff
@@ -13,5 +13,6 @@
     //Default maximum time limit for editing message(Same as server)
     public final static int DEFAULT_MAXIMUM_CONTENT_EDIT_LIMIT = 600;
     public final static boolean DEFAULT_EDITING_ALLOWED = true;
+    public static String serverUrl = null;
```

Comment:
If we declare it as final we can't assign it a new value and it needs to be initialized with some value. I can remove null as of now! Secondly I told in App preferences but @kunall17 told me to use Constants.java